config/sidekiq/sidekiq_clock.yml

```diff
@@ -0,0 +1,10 @@
+concurrency: 10
+timeout: 25
+retry: 1
+queues:
+  - clock
+
+production:
+  concurrency: <%= ENV.fetch('SIDEKIQ_CONCURRENCY', 5) %>
```

Comment:
hmm, how do you feel about a dedicated clock concurrency variable?

Reply:
since its for one worker, each worker has his own env var, do you have something else in mind?